### PR TITLE
Clarify who can start a discussion on a project

### PR DESCRIPTION
**Pull request by eeverman**
I'm proposing this change because I cannot tell who can start a new discussion.  The existing 'anyone with access to the project' is ambiguous:  Does that mean anyone who can view the project?  Can people view anonymously and start a discussion?  Is it actually only people with write permission to the project who can start discussions?

My submitted text change is my guess as to how it works.  If its wrong, I'd love to correct it and add the clarification.

### Why:

Closes [issue link]

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

File: content/discussions/quickstart.md
Comment (rcowsill) [inline content/discussions/quickstart.md:46]: How about:
```suggestion
Any logged-in user with access to a repository can create a discussion.
```

**Review by Shebu123n — APPROVED** [review]

File: content/discussions/quickstart.md
Comment (eeverman) [reply content/discussions/quickstart.md:46]: @rcowsill, how about this:
`Any logged-in user who can view the repository can create a discussion.`

**Review by yasminazaze — DISMISSED** [review]
Yasmina.zaze@icloud.com

**Comment by ramyaparimi** [comment]
@eeverman 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/discussions/quickstart.md
Comment (eeverman) [reply content/discussions/quickstart.md]: I made that change - MR should be updated.

**Comment by eeverman** [comment]
I updated this MR - I'm not sure what happens next.  The process seems stuck.

**Comment by felicitymay** [comment]
@eeverman - thanks for raising this PR 👋🏻 

> I updated this MR - I'm not sure what happens next. The process seems stuck.

Unfortunately I can't see the comment that you responded to - I guess it might have been made on a commit, rather on the pull request itself. Based on the reviewers list, I suspect that the comment wasn't left by a GitHub staff member. Currently, we require a review by a member of the GitHub docs team before PRs are merged.

I'll review your PR now and get the process started.

File: content/discussions/quickstart.md
Comment (felicitymay) [inline content/discussions/quickstart.md:46]: We usually refer to a "logged-in user" as an "authenticated user". This covers people who authenticate by logging in directly, as well as people who authenticate using other methods, like SSO.

```suggestion
Any authenticated user who can view a repository can create a discussion.
```

**Review by felicitymay — COMMENTED** [review]
Thanks for taking the time to highlight this ambiguous sentence, work out what it should say, and fix it 💖 

I'm going to make a minor change to your text, but otherwise this looks ready to merge.

**Review by felicitymay — APPROVED** [review]
The update looks good on staging: https://docs-10114--patch-2.herokuapp.com/en/discussions/quickstart#creating-a-new-discussion.

Ready to update from `main` and merge.

File: content/discussions/quickstart.md
Comment (eeverman) [reply content/discussions/quickstart.md]: Thank you!

**Pull request merged**
